src/Node/ClassStatementsGatherer.php

```diff
@@ -196,4 +211,31 @@ private function gatherNodes(Node $node, Scope $scope): void
 		$this->propertyUsages[] = new PropertyRead($node, $scope);
 	}
 
+	private function tryToApplyPropertyReads(Expr\FuncCall $node, Scope $scope): void
+	{
+		$args = $node->getArgs();
+		if (count($args) === 0) {
+			return;
+		}
+
+		$firstArgValue = $args[0]->value;
+		if (
+			!$firstArgValue instanceof Expr\Variable
+			|| $firstArgValue->name !== 'this'
+		) {
+			return;
+		}
+
+		$classProperties = $this->classReflection->getNativeReflection()->getProperties();
```

Comment:
These PHPStan failures need to be added to this file https://github.com/phpstan/phpstan-src/blob/1.9.x/build/enum-adapter-errors.neon